RDFSharpSpecflowTests/ImplicitUsings.cs

```diff
@@ -0,0 +1,6 @@
+﻿global using FluentAssertions;
+global using NUnit;
```

Comment:
A többi teszt MSTestet használ. Érdemes lenne itt is azt.

Reply:
Bár megolható lenne MSTesztet használni, hiszen a Specflow támogtaja azt is a dokumentációja szerint,  azoban az egyszerűség miatt inkább NUnit-ot használtam, mert az egyből "jön" a Specflow-val.